pandas/tests/arrays/test_integer.py

```diff
@@ -633,6 +633,15 @@ def test_astype_specific_casting(self, dtype):
         expected = pd.Series([1, 2, 3, None], dtype=dtype)
         tm.assert_series_equal(result, expected)
 
+    def test_astype_dt64(self):
```

Comment:
do we have tests with None, np.nan here as well?

Reply:
i guess those would go in the test for pd.array inference in tests.arrays.test_array, where I see a test for `[1, None]`